projects/control-service/projects/pipelines_control_service/src/main/resources/application-prod.properties

```diff
@@ -58,6 +58,8 @@ datajobs.kadmin_password=${KADMIN_PASSWORD}
 # This is the full image name of the data job builder
 datajobs.builder.image=registry.hub.docker.com/versatiledatakit/job-builder:1.1
 datajobs.deployment.dataJobBaseImage=registry.hub.docker.com/versatiledatakit/data-job-base-python-3.7:latest
+datajobs.deployment.supportedPythonVersions=${SUPPORTED_PYTHON_VERSIONS}
```

Comment:
move them to the application.properties and add default values

Reply:
They are already in application.properties I added them in prod properties, as I wasn't sure if they wouldn't be overwritten. Nevertheless, I've removed them from prod properties.